assemblies/assembly-rhdh-telemetry.adoc

```diff
@@ -10,32 +10,44 @@ As an administrator, you can disable the telemetry data collection feature based
 
 {company-name} collects and analyzes the following data:
 
+*Web Analytics (Segment)*
```

Comment:
Consider explaining what Segment and OpenTelemetry are. The words are common, but here it seems that they are tool names.

For semantics accuracy, consider also using a description list https://docs.asciidoctor.org/asciidoc/latest/lists/description/

```suggestion
Web Analytics::

Web Analytics are using the Segment tool.
```